apply-templates.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+[ -f versions.json ] # run "versions.sh" first
+
+cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+jqt='.jq-template.awk'
+if [ -n "${BASHBREW_SCRIPTS:-}" ]; then
+	jqt="$BASHBREW_SCRIPTS/jq-template.awk"
+elif [ "$BASH_SOURCE" -nt "$jqt" ]; then
+	# https://github.com/docker-library/bashbrew/blob/master/scripts/jq-template.awk
+	wget -qO "$jqt" 'https://github.com/docker-library/bashbrew/raw/9f6a35772ac863a0241f147c820354e4008edf38/scripts/jq-template.awk'
+fi
+
+jqf='template-helper-functions.jq'
+if [ "$BASH_SOURCE" -nt "$jqf" ]; then
+	wget -qO "$jqf" 'https://github.com/docker-library/bashbrew/raw/master/scripts/template-helper-functions.jq'
+fi
```

Comment:
`template-helper-functions.jq` should be added to the `.gitignore` ⬆️. The downloaded filename should start with `.` so that it becomes a "hidden" file like the awk script.

I think we ought to have a similar `BASHBREW_SCRIPTS` check like the awk script does. The env var is set by the [`docker-library/bashbrew` action](https://github.com/docker-library/bashbrew/blob/a9fce37a003383b4fbaabb29ac810a8d80964e76/action.yml#L18) so that the scripts don't have to be downloaded when that repo is already checked out (used in [`ci.yml`](https://github.com/docker-library/redis/blob/92c70b44fbc5b27df1a14f90a0bd3e1cda46d752/.github/workflows/ci.yml#L22)).

The download of `template-helper-functions.jq` should use a specific commit like the awk script does.

(or maybe you were planning to add these bits when adding the `sbom` bits in another PR, so this `if` can be dropped here and improved over there).